### DISCUSSION TITLE
UCI Architecture Review

### DISCUSSION THREAD

**Discussion by aayushkedia95** [Samagra-Development/SamagraX-community#33] [Installation and Configuration]
@Krishnaj20 @vrayulu, we were hoping to close UCI detailed architecture review this week (over 2 discussions). Would it be possible to get on a call tomorrow, first half?

cc-  @ChakshuGautam

**Comment by Krishnaj20** [comment]
@aayushkedia95 - @vrayulu is not available this week due to personal exigency. Pls plan for this session next Monday/Tuesday.

**Reply by aayushkedia95** [reply]
Noted, @Krishnaj20. We will schedule this next week.
